android/app/src/main/java/org/smartregister/fhircore/fragment/PatientListFragment.kt

```diff
@@ -111,6 +114,15 @@ class PatientListFragment : Fragment() {
       }
     )
 
+    btnOverdue.setOnCheckedChangeListener { buttonView, isChecked ->
+      patientListViewModel.searchResults(
```

Comment:
I think this is simply supposed to set a sharedpreference boolean that can be later accessed here https://github.com/opensrp/fhircore/pull/194/files#diff-14402c85a380eedd89160390f84f7c082323e0c97d31388c3d3c43e278d53a94R95.

Reply:
fixed. now using SP